kornia/filters/blur_pool.py

```diff
@@ -202,3 +202,47 @@ def _max_blur_pool_by_kernel2d(
     # blur and downsample
     padding: Tuple[int, int] = _compute_zero_padding((kernel.shape[-2], kernel.shape[-1]))
     return F.conv2d(input, kernel, padding=padding, stride=stride, groups=input.size(1))
+
+
+def edge_aware_blur_pool2d(
+    input: torch.Tensor,
```

Comment:
```suggestion
    input: Tensor,
```
`from kornia.core import Tensor`